harborclient/harborclient.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 
-import requests
-import simplejson
+# for inner
+import json
 import logging
+# for thirdparty
```

Comment:
I'm not sure if we need this. The third party dependencies will be listed in `requirements.txt`.